scene/gui/control.cpp

```diff
@@ -436,6 +436,7 @@ void Control::_resize(const Size2 &p_size) {
 //moved theme configuration here, so controls can set up even if still not inside active scene
 
 void Control::add_child_notify(Node *p_child) {
+	CanvasItem::add_child_notify(p_child);
```

Comment:
I wonder if this should be called first or last as a general rule (I don't have the answer).